### PR TITLE
Inputs

### PR DESCRIPTION
**Pull request by SumitSaundadkar**

### CODE REVIEW

File: components/input.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+.input_bar {
+    display: flex;
+    flex-direction: row;
+    width: 80%;
+    background-color: var(--offline-color);
+    padding: 10px;
+    margin: 10px;
+    align-items: center;
+    justify-content: space-around;
+   border-radius: 10px;
```
Comment (anshulraheja) [inline components/input.css:16]: The border-radius is being applied on the div and not on the input box

File: components/input.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
+
+ .input{
+     width: 80%;
+     padding: 10px;
+     height: 1rem;
```
Comment (anshulraheja) [inline components/input.css:5]: Redundant code. The default height of input is 1rem only.

**Review by anshulraheja — APPROVED** [review]
Clean UI

File: components/input.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  }
+.input_bar {
+    display: flex;
+    flex-direction: row;
```
Comment (dhimanaishwarya) [inline components/input.css:9]: not required , by default flex-direction is row

File: components/input.css (hunk 1 above)
Comment (j24m) [reply components/input.css:16]: +1

File: components/input.css (hunk 2 above)
Comment (j24m) [reply components/input.css:5]: +1

File: components/input.css (hunk 3 above)
Comment (j24m) [reply components/input.css:9]: +1

**Review by j24m — COMMENTED** [review]
The changes suggested by others can be done and then you are good to merge, btw great work, clean and readable code.:)

**Review by m26unkwn — APPROVED** [review]

File: components/input.css (hunk 1 above)
Comment (SumitSaundadkar) [reply components/input.css:16]: thanks updated

File: components/input.css (hunk 2 above)
Comment (SumitSaundadkar) [reply components/input.css:5]: updated

File: components/input.css (hunk 3 above)
Comment (SumitSaundadkar) [reply components/input.css:9]: updated

**Pull request merged**